### DISCUSSION TITLE
Workspace Extensions

### DISCUSSION THREAD

**Discussion by dreamy32** [spaciousejar/learn-git-with-me#5] [Q&A] [answered]
I would like to know the extensions you use for this project and/or regularly. This could really help for productivity :)

**Comment by spaciousejar** [comment, marked as answer]
Please read this " https://github.com/spaciousejar/learn-git-with-me/blob/main/README.md ."

**Comment by spaciousejar** [comment]
https://github.com/spaciousejar/learn-git-with-me/blob/main/CONTRIBUTING.md
